src/terminologySectionPanel/codesSection/codesSubSection/common/EditCodeDetails.tsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import { TextField } from "@madie/madie-design-system/dist/react";
+import { Checkbox, FormControlLabel } from "@mui/material";
+import tw from "twin.macro";
+import "styled-components/macro";
+
+export default function EditCodeDetails({ selectedCodeDetails }) {
```

Comment:
This method can be a little consolidated with simpler code instead of different return statements?

Reply:
thank you, I made the change